core/tests/test_utils.py

```diff
@@ -143,12 +158,26 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """ Supply a test string that has unicode characters in it.
+
+        Args:
+            suffix: str. The suffix to UNICODE_TEST_STRING.
+
+        Returns:
+            str. A string UNICODE_TEST_STRING with suffix 'suffix'.
+        """
         return '%s%s' % (self.UNICODE_TEST_STRING, suffix)
 
     def setUp(self):
+        """Initializes the fixture for the test suite.
+        Subclasses of TestBase should override this method.
+        """
         raise NotImplementedError
 
     def tearDown(self):
+        """Cleans up the fixture after the test runs.
+        Subclasses of TestBase should override this method.
+        """
         raise NotImplementedError
 
     def _assert_validation_error(self, item, error_substring):
```

Comment:
You don't need to document `self`, but there are other args here besides self...